youtube_dl/extractor/playtvak.py

```diff
@@ -0,0 +1,127 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import (
+    compat_urlparse,
+    compat_urllib_parse,
+)
+from ..utils import ExtractorError
+
+
+def _extract_json(code):
+    return re.sub(
+        r'(?s)^VideoPlayer.data\("", ({.*})\);?\s*?(?://[^\n]*)*$', r'\1', code)
```

Comment:
Copied from [`strip_jsonp` function](https://github.com/rg3/youtube-dl/blob/master/youtube_dl/utils.py#L1622). It should catch and drop anything after the interesting json.